cli/cmd/secrets.go

```diff
@@ -19,19 +19,19 @@ package cmd
 import (
 	"fmt"
 	"io"
+	"io/ioutil"
 	"os"
 
 	"github.com/spf13/cobra"
 
 	"github.com/docker/compose-cli/api/client"
 	"github.com/docker/compose-cli/api/secrets"
 	"github.com/docker/compose-cli/formatter"
+	"github.com/docker/docker/pkg/system"
 )
 
 type createSecretOptions struct {
 	Label       string
-	Username    string
-	Password    string
```

Comment:
This matches the existing CLI's secrets implementation which is flat (i.e.: secret name maps to a value) but I think it will break pulling private images where we need a dictionary with username and password.